script.100fm/resources/settings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+    <category>
+        <setting label="10000" type="number" id="channel" default="0" visible="false"/>
```

Comment:
Hi @phil65 , first of all - thank you for understanding the situation. I tried to to remove the settings file and fix the code which caused it to crash, but without the settings file - settings are just not saved. You might want to try it yourself and see that "now playing" labels not updating, for example. So unless I'm missing something, can we leave it as is?
Thanks.